package.json

```diff
@@ -16,6 +16,7 @@
     "clean": "rm -rf node_modules */**/node_modules */**/.turbo"
   },
   "devDependencies": {
+    "@prettier/plugin-xml": "^2.2.0",
```

Comment:
why do we need this? 😅

Reply:
Heheh should have added some more context to that commit 🤡 We need it to format svgs:))